telco-hub/configuration/reference-crs/optional/cert-manager/consoleCertificate.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: console-certificate
+  namespace: cert-manager-operator
+spec:
+  commonName: selfsigned-ca.dns.name
+  isCA: true
```

Comment:
I believe this should be false for these certs (here and below)

Reply:
Changed to false.